spec/models/intake_spec.rb

```diff
@@ -1125,4 +1124,44 @@
       it { expect(intake.triaged_from_stimulus?).to be_falsey }
     end
   end
+
+  describe "#must_have_documents" do
+    let(:intake) { create(:intake) }
+
+    before do
+      allow(DocumentNavigation).to receive(:document_types_for_intake).with(intake).and_return(["1095-A", "Selfie", "W-2"])
```

Comment:
It's great that this PR mocks out `DocumentNavigation` -- really helps with clarity around separation of concerns.